packages/ses/README.md

```diff
@@ -1,17 +1,30 @@
-# Secure EcmaScript (SES)
+# Secure ECMAScript (SES)
 
-Secure EcmaScript (SES) is a frozen environment for running EcmaScript
+Secure ECMAScript (SES) is an execution environment that provides fine-grained
+sandboxing with Compartments.
+
+* **Compartments** Compartments are separate execution contexts: each one has its own global object and global lexical scope.
+* **Frozen realm** Compartments share their intrinsics to avoid identity discontinuity. By freezing the intrinsics, SES removes programs abilities to interfere with each other.
```

Comment:
```suggestion
* **Frozen realm** Compartments share their intrinsics to avoid identity discontinuity. By freezing the intrinsics, SES removes programs' abilities to interfere with each other.
```

Reply:
or
```suggestion
* **Frozen realm** Compartments share their intrinsics to avoid identity discontinuity. By freezing the intrinsics, SES prevents programs from interfering with each other.
```
or
```suggestion
* **Frozen realm** Compartments share their intrinsics to avoid identity discontinuity. By freezing the intrinsics, SES protects programs from each other.
```